ansible_collections/arista/avd/roles/eos_designs/docs/input-variables.md

```diff
@@ -154,12 +154,12 @@ AVD provides the capability to customize your node types, supporting a variety o
     ```yaml
     node_type_keys:
 
-      - key: spine
+    - key: spine
```

Comment:
These changes break stuff. I am not sure you intended to change all these things in this file, so please revert what was done by mistake.

Reply:
these changes were not intended, maybe one of my plugins removed spaces and I accidentially saved them... I copied the lines from the file in the dev branch, hope it is all sorted now.